templates/geosafe/analysis/create.html

```diff
@@ -520,7 +520,13 @@ <h4 class="modal-title" id="myModalLabel">{% trans "Upgrade Your Browser" %}</h4
             });
 
             {# Zoom first to global world view #}
-            zoom_to_box(map, [0, 45, 0, -45]);
+            {%  if analysis and analysis.user_extent %}
+                user_extent_string = "{{ analysis.user_extent }}";
+                extent_after_run = user_extent_string.split(",");
+                zoom_to_box(map, extent_after_run);
```

Comment:
Have you check this? 
I don't think this is going to work, because after split, the list contains string, no?
You could use `"[" + {{ analysis.user_extent }} + "]"`, because user_extent were already in the bbox format.